test/DataFixtures/Ticket/EmailData.php

```diff
@@ -23,7 +23,7 @@ class EmailData extends BaseModelData
         'delete_downloaded' => null,
         'created_at' => 0,
         'updated_at' => 0,
-        'oauth' => 1,
+        'oauth' => '1',
```

Comment:
Better if this is a random string, it's essentially an encrypted string like a password.

Reply:
this doesn't really matter, there's no logical value beyond the type being checked